src/components/app/exporters/YnabAccountMappingTable.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <v-simple-table dense>
+    <template v-slot:default>
+      <thead>
+        <tr>
+          <th>Account number</th>
+          <th>Ynab account id</th>
+          <th>Delete</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr
+          v-for="accountNumberAndId in accountNumbersToYnabAccountIds"
+          :key="accountNumberAndId.accountNumber"
+        >
+          <td>
+            <v-text-field
+              v-model="accountNumberAndId.accountNumber"
+              full-width
+              dense
+              :rules="[rules.required]"
+              @keydown="$emit('mappingChanged')"
+            />
+          </td>
+          <td>
+            <v-text-field
+              v-model="accountNumberAndId.ynabAccountId"
+              full-width
+              dense
+              :rules="[rules.required]"
+              @keydown="$emit('mappingChanged')"
+            />
+          </td>
+          <td>
+            <v-btn
+              icon
+              color="red"
+              @click="deleteAccountMapping(accountNumberAndId.accountNumber)"
+            >
+              <v-icon>mdi-delete</v-icon>
+            </v-btn>
+          </td>
+        </tr>
+      </tbody>
+    </template>
+  </v-simple-table>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { required } from '@/components/shared/formValidations';
+
+export default Vue.extend({
+  name: 'YnabAccountMappingTable',
+  props: {
+    accountNumbersToYnabAccountIds: {
+      type: Array,
+      required: true
+    }
+  },
+  setup(_props) {
+    return {
+      deleteAccountMapping(accountNumberToDelete) {
+        // @ts-ignore
+        this.$emit('deleteAccountMapping', accountNumberToDelete);
+      },
```

Comment:
You have a bug here:

1. Add new *Account to Ynab* item.
2. Set value for `Account number`.
3. Try to delete this item.

Also, the + button is stuck after that (and maybe without these steps)

![searchlist2](https://user-images.githubusercontent.com/17686879/93916525-5bd2c600-fd12-11ea-9cd8-7183209ed454.gif)

Reply:
I don't research this deeply. Just a question (I'm not sure how *Two Way Binding* works, I'm always googling it), why you're using these `events` and not just use `v-model`?

As I remember, `v-model` is shortcut for `value` prop and `input` event, so you can get an array of pairs, and add/remove/change it inside `YnabAccountMappingTable`, and `emit` it in any change.

Reply:
Solved. I don't know if my solution is better in all the aspects, but I think we need to use [`v-model`](https://vuejs.org/v2/guide/components.html#Using-v-model-on-Components).

Then I had a lot of problems, I worked on this a day, so first I managed to get it to work, and now we have something to improve.